### PR TITLE
fix: bug in login with --api-token flag

### PR DESCRIPTION
**Pull request by jarqvi**

### CODE REVIEW

File: src/commands/account/add.ts [hunk 1]
```diff
@@ -70,24 +76,36 @@ export default class AccountAdd extends Command {
...
+                headers: { Authorization: undefined },
+              })
+              .json<IAccount>();
+            return data;
+          } else {
+            let user = await this.getMe(flags);
+
+            user.api_token = flags['api-token'];
+
+            return user;
```
Comment (mhemrg) [inline src/commands/account/add.ts:95]: Why do you repeat this logic here again?

**Comment by mhemrg** [comment]
We shouldn't ask for an email address when `--api-token` has been provided.

**Comment by jarqvi** [comment]
It doesn't ask such a question and the code works fine, please check again.

**Comment by jarqvi** [comment]
test

**Comment by jarqvi** [comment]
test

**Pull request merged**